unitTests/testTensorOpsEigen.cpp

```diff
@@ -189,12 +189,13 @@ class TestEigendecomposition : public ::testing::Test
 
       // Construct a diagonal matrix of the eigenvalues
       FLOAT lambda[ tensorOps::SYM_SIZE< M > ];
```

Comment:
Shouldn't this be `FLOAT lambda[M]`?

Reply:
It needs to be a symmetric matrix to leverage the `tensorOps` operations.

Reply:
the lambda's are the eigenvalues...of which there are 3 for a 3x3, whether it is symmetric or not.

Reply:
On line 202 I call
```
tensorOps::AikSymBklAjl< M >( matrices[ i ], Q, lambda );
```
So `lambda` needs to be in symmetric notation for that call to work.